Pony/include/pony/Address.hpp

```diff
@@ -6,37 +6,41 @@
 /* Original story "Networking for Game Programmers" by Glenn Fiedler
  */
 
-#ifndef PONY_ADDRESS_HPP
-#define PONY_ADDRESS_HPP
-
-#include <pony/SharedMacros.hpp>
+#ifndef ___PONY__ADDRESS_HPP___
```

Comment:
We have another notations for header guards
like PONY_ADDRESS_HPP

Reply:
ok